src/cluster/generated/proto/kvpb/kv.proto

```diff
@@ -36,7 +36,7 @@ message KeyValueUpdateResult {
 message QueryLimits {
 	QueryLimit maxRecentlyQueriedSeriesBlocks = 1;
 	QueryLimit maxRecentlyQueriedSeriesDiskBytesRead = 2;
-	QueryLimit maxRecentlyQueriedSeriesDiskRead = 3;
+    reserved 3;
```

Comment:
nit: Whitespace doesn't seem to match the previous two lines? Probably a tabs/spaces thing?

Reply:
hmm weird. thanks